root/relationship/linkattributetype/RelationshipAttributeTypeInUse.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * @flow
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import Layout from '../../layout';
+
+const RelationshipAttributeTypeInUse = ({type}: {type: LinkAttrTypeT}) => (
+  <Layout fullWidth page="in-use" title={l('Relationship attribute in use')}>
+    <div className="content">
+      <h1>{l('Relationship attribute in use')}</h1>
+      <p>
+        {exp.l(
+          `The relationship attribute type “{type}” can not be deleted
+           because it is is still in use.`,
```

Comment:
Not your fault, but can we fix this string while we're at it? :P

(I prefer contractions because they're much easier to read and sound less weirdly formal, but in any case "can not" and "is is" are wrong.)

```suggestion
          `The relationship attribute type “{type}” can’t be deleted
           because it’s still in use.`,
```